metaflow/plugins/kfp/kfp_pod_decorator.py

```diff
@@ -0,0 +1,27 @@
+from metaflow.decorators import StepDecorator
+
+
+class PodAnnotationDecorator(StepDecorator):
+    """
+    Add a single pod annotation. This decorator can be used multiple time per step.
+    Repeated adding assigning value under same name will overwrite previous value.
+    """
+
+    name = "pod_annotation"
+    defaults = {
+        "name": None,
```

Comment:
Do we want to call these `key`/`value` as that's a more natural naming scheme to me, anyone else have any thoughts?

Reply:
Didn't see Hari's comment, closing 😅  https://github.com/zillow/metaflow/pull/59#discussion_r580632968

Reply:
`key`/`value` sounds good to me.